Assets/Scripts/Audio/FmodEventHandler.cs

```diff
@@ -0,0 +1,335 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
+/// Class that holds the information for playing and updating parameters on a specific fmod event, for both tonal and atonal events.
+/// The actual playing of the fmod event will occur through the fmod event pool based on the information stored in this class.
+/// Any time we need to play an fmod event, it should be accessed through a gameobject with an instance of this class.
+/// </summary>
+public class FmodEventHandler : MonoBehaviour
+{
+    public string sfxEventName = "";
```

Comment:
Usually I'd agree with you, but in this case the variables are ordered the way they are because sfxEventName and sfxEventVolume are directly tied to one another in terms of the fmod event they reference. Since they're exposed in the editor, I'd prefer them to be in this order.